Src/Support/Google.Apis.Auth/Google.Apis.Auth.csproj

```diff
@@ -43,15 +43,17 @@ Supported Platforms:
     <Reference Include="System.Net.Http" />
   </ItemGroup>
 
-  <!-- build and include the (empty) platformservices dll in package; required for backward compatibility -->
+  <!-- build and include the (empty) platformservices dll in package; required for binary backward compatibility -->
   <!-- There must be a better way of doing this... -->
   <Target Name="IncludePlatformServices" BeforeTargets="GenerateNuspec" Condition="'$(OS)' == 'Windows_NT'">
     <MSBuild Projects="..\Google.Apis.Auth.PlatformServices\Google.Apis.Auth.PlatformServices.csproj" />
     <Copy SourceFiles="..\Google.Apis.Auth.PlatformServices\bin\$(Configuration)\netstandard1.3\Google.Apis.Auth.PlatformServices.dll" DestinationFolder="$(OutputPath)netstandard1.3" />
     <Copy SourceFiles="..\Google.Apis.Auth.PlatformServices\bin\$(Configuration)\net45\Google.Apis.Auth.PlatformServices.dll" DestinationFolder="$(OutputPath)net45" />
   </Target>
   <ItemGroup Condition="'$(OS)' == 'Windows_NT'">
-    <!-- netstandard2.0 not required; it's a newly supported framework, so binary compatilibity is not needed. -->
+    <!-- netstandard2.0 not required. To use the netstandard2.0 assemblies in this packages requires a re-build of the dependant code,
```

Comment:
I suspect this argument doesn't hold, much as it would be nice if it did. Consider:

- An application that targets netcoreapp2.0
- The application depends directly on Google.Apis.Auth v1.35 and SomePackage
- SomePackage targets netstandard1.5 and depends on Google.Apis.Auth v1.30 (or some other older version)

In that situation, I *think* we end up with:

- App.exe
- SomePackage.dll
- Google.Apis.Auth.dll v1.35.0

... but SomePackage.dll has a reference in its IL to Google.Apis.Auth.PlatformServices.dll.

Bang at runtime.

Does this seem plausible? We can test it if necessary.

Reply:
Yes, I'm pretty sure you're right. I'll change this PR to include the PlatformServices dll in the netstandard2.0 package.

Reply:
Fixed